packages/runtime/src/enhance.d.ts

```diff
@@ -1,2 +1,2 @@
 // @ts-expect-error stub for re-exporting generated code
-export { enhance } from '.zenstack/enhance';
+export { auth, enhance } from '.zenstack/enhance';
```

Comment:
Thanks for making this fix @israelins85 !

I believe the generated `auth` namespace also needs to be exported. 

https://github.com/zenstackhq/zenstack/blob/64e0c98acc4cd018c5169f3b325a3d4b617290ef/packages/schema/src/plugins/enhancer/enhance/auth-type-generator.ts#L109